pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -859,8 +872,19 @@ pyramid_oereb:
         get_symbol: pyramid_oereb.standard.hook_methods.get_symbol
         get_symbol_ref: pyramid_oereb.standard.hook_methods.get_symbol_ref
       law_status:
-        in_force: inForce
-        running_modifications: runningModifications
+        inKraft: inKraft
+        AenderungMitVorwirkung: AenderungMitVorwirkung
+        AenderungOhneVorwirkung: AenderungOhneVorwirkung
+      # Example of how sub_themes sorting can be activated (uncomment to enable):
+      #sub_themes:
+      #  sorter:
+      #    module: pyramid_oereb.contrib.print_proxy.sub_themes.sorting
+      #    class_name: ListSort
+      #    params:
+      #      list:
+      #        - SubTheme3
+      #        - SubTheme2
+      #        - SubTheme1
```

Comment:
This part should be moved to the Land Use Plan section as it is basically the only one for now that allows subthemes.
But I thought Cécile already moved it in an earlier PR? (please check and delete or move this comment)

Reply:
I probably accidentally copied this part here. Just removed it